client/src/components/forms/SignupForm.js

```diff
@@ -84,15 +85,10 @@ const SignupForm = () => {
           validate: value => value === getValues('password')
         })}
       />
-      {errors.matchPassword && <span>Password does not match</span>}
-
-      {error &&
-        <p className="error">
-          <strong>Error: </strong>
-          {error}
-        </p>
-      }
```

Comment:
this error triggers when there is an error on the backend, that's why it has a prefacing *Error: *. I think it would be nice to have this styled a bit differently